wpilibj/src/athena/java/edu/wpi/first/wpilibj/IterativeRobot.java

```diff
@@ -307,5 +307,6 @@ public void testPeriodic() {
       System.out.println("Default IterativeRobot.testPeriodic() method... Overload me!");
       m_tmpFirstRun = false;
     }
+    Timer.delay(0.001);
```

Comment:
Actually, I think it would be better to remove these waits from IterativeRobot. In SampleRobot they make sense, however in Iterative they just cause an unnecessary context switch, as WaitForData will immediately yield anyway.